app/src/main/kotlin/com/wire/android/notification/broadcastreceivers/SummaryNotificationDismissReceiver.kt

```diff
@@ -39,7 +38,7 @@ class SummaryNotificationDismissReceiver : BroadcastReceiver() {
                 } else {
                     val currentSession = coreLogic.globalScope { session.currentSession() }
                     if (currentSession is CurrentSessionResult.Success) {
-                        coreLogic.getSessionScope(currentSession.authSession.tokens.userId)
+                        coreLogic.getSessionScope(currentSession.authSession.session.userId)
```

Comment:
I'm just being nitpicky, but I feel like the name 'authSession.session' is wrong 😅
maybe we can revert it back to tokens